Photo-Sharing-Sample/PhotoSharingSample.xcodeproj/project.pbxproj

```diff
@@ -703,7 +703,7 @@
 			repositoryURL = "https://github.com/aws-amplify/amplify-ios";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = 1.12.0;
+				minimumVersion = 1.29.0;
```

Comment:
1.29.0 includes the CPK changes of `amplify codegen models`. We can also bump this to 2.3.2 but requires more code changes than the scope of this PR. I tried 2.3.2 and saw some code changes needed for the Storage API calls, since they are using Async Await APIs in 2.0.0 and up